code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -64,27 +64,28 @@
 
 /mob/living/carbon/smoke_contact(obj/effect/particle_effect/smoke/S)
 	. = ..()
-	var/protection = .
+	var/protections = ..()
```

Comment:
You're calling parent twice

Reply:
Do I call it in the var or just the . = ..(), Im not that good with byond code.